<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BotonRegistroVideo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver Registros videos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BotonRegistoAlertas">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver Registro de Alertas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BotonRegistroAcceso">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver Registro de acceso</string>
   </property>
  </widget>
  <widget class="QLabel" name="MostrarCamara">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
